styles/modules/Speaker.module.css

```
@@ -10,68 +11,110 @@
   font-size: 4.8rem;
 }
 
-.speakerContainer span {
-  margin-top: 40px;
-  font-size: 2.4rem;
-  color: #afafaf;
+.speakerContainer .speakerFlexContainer {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  gap: 2rem;
+}
+
+.speakerContainer .speakerFlexContainer .mainSession {
+  display: flex;
+  flex-direction: column;
+  width: 48.4rem;
+  height: auto;
 }
 
-.speakerContainer .speakerGridContainer > div:nth-child(2) {
-  margin-top: 20rem;
+.speakerContainer .speakerFlexContainer .talkSession {
+  width: 48.4rem;
+  height: auto;
 }
 
-.speakerInfoContainer h3 {
-  font-size: 2.4rem;
-  color: #afafaf;
+.speakerFlexContainer .speakerSessionContainer {
+  height: auto;
+  margin-top: 2rem;
+  background-color: #191919;
+  padding: 2.4rem 3rem;
+  border-radius: 8px;
 }
 
-.speakerGrid {
-  color: white;
-  font-size: 1.6rem;
-  height: 50%;
-  display: grid;
-  grid-template-rows: repeat(auto-fill, minmax(10rem, 1fr));
-  grid-template-columns: repeat(auto-fill, minmax(20rem, 1fr));
-  text-align: center;
-  margin-top: 4rem;
-  gap: 10rem;
+.speakerSessionContainer .mainIntroTitle {
+  width: 100%;
+  height: 2.4rem;
+  font-weight: bold;
+  font-size: 1.7rem;
+  color: #fe00b8;
+  border-bottom: 0.1rem solid #fe00b8;
+}
+
+.speakerSessionContainer .talkIntroTitle {
+  width: 100%;
+  height: 2.4rem;
+  font-weight: bold;
+  font-size: 1.7rem;
+  color: #1920ff;
+  border-bottom: 0.1rem solid #1920ff;
+}
+
+.speakerSessionContainer .sessionContainer {
+  font-weight: bold;
+  line-height: 2.4rem;
+}
+
+.speakerSessionContainer .sessionContainer h5 {
+  font-weight: bold;
+  font-size: 2.2rem;
+  margin-top: 2.4rem;
+}
+
+.speakerSessionContainer .sessionContainer div {
+  font-weight: 400;
+  font-size: 1.7rem;
+  margin-top: 0.8rem;
+  white-space: pre-line;
 }
 
-.speakerProfileContainer {
+.speakerSessionContainer .speakerInfo {
+  margin-top: 7.2rem;
   display: flex;
-  flex-direction: column;
-  justify-content: center;
-  align-items: center;
+  gap: 1.6rem;
+  color: #cfcfcf;
+  font-weight: bold;
 }
 
-.speakerGrid .speakerProfileContainer .speakerProfileImageBox {
-  position: relative;
-  border-radius: 10px;
-  width: 18rem;
-  height: 18rem;
+.speakerSessionContainer .speakerInfo .speakerProfileImg {
+  width: 10rem;
+  height: 10rem;
```

```suggestion
  min-width: 10rem;
  min-height: 10rem;
```
모바일 적용시 이미지 값이 변하지 않게 하기 위해 min값으로 설정.